### PR TITLE
chore(ci): add scripts to test against multiple releases of AngularJS

### PR DESCRIPTION
**Pull request by matsko**

### CODE REVIEW

File: scripts/download-and-test-version.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+VERSION=$1
```
Comment (jelbourn) [inline scripts/download-and-test-version.sh]: This file could use a comment explaining what it does / why it exists.

File: scripts/find-max-version.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+var exec = require('child_process').exec;
```
Comment (jelbourn) [inline scripts/find-max-version.js]: This file could use a comment explaining what it does / why it exists.

File: scripts/download-and-test-version.sh (hunk 1 above)
Comment (matsko) [inline scripts/download-and-test-version.sh]: This file was actually left over by accident.

File: scripts/test-versions.sh [hunk 3]
```diff
@@ -0,0 +1,93 @@
+CDN="https://code.angularjs.org"
```
Comment (jelbourn) [inline scripts/test-versions.sh]: This file could use a comment explaining what it does / why it exists.

**Comment by jelbourn** [comment]
Overall looks good, though I generally like more explanation sprinkled throughout explaining why you're doing certain things.

**Comment by jelbourn** [comment]
@robertmesserle to take a look also

**Comment by matsko** [comment]
@jelbourn @robertmesserle please have a look again. I've added docs and input validation.

**Comment by jelbourn** [comment]
LGTM

**Comment by ThomasBurleson** [comment]
@matsko - Let's wait until @robertmesserle  reviews before I merge. Thx

**Comment by robertmesserle** [comment]
LGTM

**Comment by matsko** [comment]
Landed as https://github.com/angular/material/commit/1a0fc6b03abc197691ff1eff4e803ab2133d3c57

**Pull request closed without merging**